assets/js/blocks/stock-filter/edit.tsx

```diff
@@ -51,10 +65,62 @@ const Edit = ( {
 							} )
 						}
 					/>
-				</PanelBody>
-				<PanelBody
-					title={ __( 'Settings', 'woo-gutenberg-products-block' ) }
-				>
+					<ToggleGroupControl
+						label={ __(
+							'Allow selecting multiple options?',
+							'woo-gutenberg-products-block'
+						) }
+						value={ selectType || 'multiple' }
+						onChange={ ( value: string ) =>
+							setAttributes( {
+								selectType: value,
+							} )
+						}
+						className="wc-block-attribute-filter__multiple-toggle"
+					>
+						<ToggleGroupControlOption
+							value="multiple"
+							label={ __(
+								'Multiple',
+								'woo-gutenberg-products-block'
+							) }
+						/>
+						<ToggleGroupControlOption
+							value="single"
+							label={ __(
+								'Single',
+								'woo-gutenberg-products-block'
+							) }
+						/>
```

Comment:
You know what? Thinking of this, this should just be a toggle switch, I believe. It's basically “Allow selecting multiple options?”: yes / no.

Reply:
Good catch and I totally agree! I took it from Filter to keep them aligned. Having said that and keeping in mind I did the same for Filter by Rating as well ([PR](https://github.com/woocommerce/woocommerce-blocks/pull/7771)), do you mind if I create an issue and change the settings in all three places at once, once both Filters (by Rating and by Stock Status) are merged?